lib/yard/docs_helper.rb

```diff
@@ -6,7 +6,7 @@ module DocsHelper
     def one_of(enumerable, lower: false, sort: true)
       # Sort the array if requested
       if sort
-        compare = ->(a, b) { a.class == b.class ? a <=> b : a.class.to_s <=> b.class.to_s }
+        compare = ->(a, b) { a.class.instance_of?(b.class) ? a <=> b : a.class.to_s <=> b.class.to_s }
```

Comment:
It seems like this broke the one of sorting? I'm not sure why though

Reply:
Ah ok my bad, this should be `a.instance_of?(b.class)`. I think I misread the rubocop output.